core_backend/app/llm_call/process_output.py

```diff
@@ -341,7 +346,6 @@ async def _generate_tts_response(
         logger.warning("TTS generation skipped due to missing LLM response.")
         return response
 
```

Comment:
Must have lost some context, why are we not checking for the language?

Reply:
here we are just checking if the LLM gave us a language or not as in the `IdentifiedLanguage` should never be none, then we map the appropriate codes inside the speech function itself